testing/file_test/test_file.cpp

```diff
@@ -133,30 +174,52 @@ static auto ReplaceLspKeywordAt(std::string* content, size_t keyword_pos,
   llvm::StringRef body = body_start.take_front(body_end);
   auto [method_or_id, extra_content] = body.split(":");
 
-  // Form the JSON.
-  std::string json = llvm::formatv(R"({{"jsonrpc": "2.0", "{0}": "{1}")",
-                                   method_or_id_label, method_or_id);
-  if (use_call_id) {
-    // Omit quotes on the ID because we know it's an integer.
-    json += llvm::formatv(R"(, "id": {0})", ++lsp_call_id);
-  }
+  llvm::json::Value parsed_extra_content = nullptr;
   if (!extra_content.empty()) {
-    json += ",";
-    if (extra_content_label.empty()) {
-      if (!extra_content.starts_with("\n")) {
-        json += " ";
-      }
-      json += extra_content;
-    } else {
-      json += llvm::formatv(R"( "{0}": {{{1}})", extra_content_label,
-                            extra_content);
+    std::string bracketed_extra_content =
+        llvm::formatv("{{{0}}", extra_content);
+    auto parse_result = llvm::json::parse(bracketed_extra_content);
+    if (auto err = parse_result.takeError()) {
+      return ErrorBuilder() << "Error parsing extra content: " << err;
+    }
+    parsed_extra_content = std::move(*parse_result);
+    CARBON_CHECK(parsed_extra_content.kind() == llvm::json::Value::Object);
```

Comment:
Similarly seems easy for a fuzzer to hit this, maybe make it an error?

Reply:
Don't *think* this one can be hit by the fuzzer, we wrap the json in {} tokens so either a. we get an object back or b. the parse fails.

Reply:
Maybe it'd help to rename `bracketed_extra_content` to `extra_content_as_object`? The use of `{}` to trigger object evaluation seems subtle, but I agree it shouldn't be an issue.